### ISSUE TITLE
`ssh` add option to specify the worker username for the `ssh` command 

### ISSUE THREAD

**Issue by petersutter** [gardener/gardenctl-v2#322]
**What would you like to be added**:
Add an option for gardenctl in order to override the username for the shoot worker. E.g. `--node-username=admin` 

**Why is this needed**:
If the execute cloud config downloader script did not run through successfully, the gardener user is most likely not created and thus a login with this user will fail. In this case the user should try with the admin user as fallback.

**Comment by petersutter** [comment]
related https://github.com/gardener/gardenctl-v2/issues/322

**Comment by tedteng** [comment]
one question does the `admin` user only create on OpenStack as the default user? 
Recently I was looking for a feasible solution on this ticket starting from GCP.   I checked `cat /etc/passwd` don't find the `admin` user.  and most of the existing user from os is `/usr/sbin/nologin`  which is unable to use as  `ssh` account. or the `admin` name is just a substitute in here, could be any name that we created later?
